### PR TITLE
style-guide: Provide a general overview of appropriate content.

### PR DESCRIPTION
**Pull request by flexibeast**
Addresses [this comment](https://github.com/void-linux/void-docs/issues/190#issuecomment-613205828) by @bobertlo.

### CODE REVIEW

**Comment by ericonr** [comment]
Very explicit, I like it. Now I just need to remember to read it :D

File: src/contributing/void-docs/style-guide.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ This style guide outlines the standards for contributing to the
 [mdBook](https://rust-lang-nursery.github.io/mdBook/) stored in the
 [void-docs](https://github.com/void-linux/void-docs/) repository.
 
+## General
+
+Step-by-step instructions containing 'magic' commands for copying-and-pasting
+are strongly discouraged. Users are expected to read the canonical documentation
```
Comment (bobertlo) [inline src/contributing/void-docs/style-guide.md:12]: This could probably be qualified with "outside of the installation guides section" just to be honest. We need to be careful there still but in the end, we should still provide a set of commands that can be combined to get Void installed.

File: src/contributing/void-docs/style-guide.md [hunk 2]
```diff
@@ -6,6 +6,19 @@ This style guide outlines the standards for contributing to the
...
+(e.g. man pages) for individual programs to understand how to use them, rather
+than relying on blindly copying-and-pasting.
```
Comment (bobertlo) [inline src/contributing/void-docs/style-guide.md:14]: blindly might be too much? we want to keep this friendly :)

Comment (flexibeast) [reply src/contributing/void-docs/style-guide.md:14]: Sure. :-) Removed.

File: src/contributing/void-docs/style-guide.md [hunk 3]
```diff
@@ -6,6 +6,19 @@ This style guide outlines the standards for contributing to the
 [mdBook](https://rust-lang-nursery.github.io/mdBook/) stored in the
 [void-docs](https://github.com/void-linux/void-docs/) repository.
 
+## General
+
+Step-by-step instructions containing 'magic' commands for copying-and-pasting
```
[the remark this answers is not in the document]
Comment (flexibeast) [reply src/contributing/void-docs/style-guide.md:11]: How about: "Although there will always be cases where command listings are appropriate, the contents of the Handbook should be written primarily in prose"?

i use 'prose' here rather than 'English' in case people think we mean "the language of the Handbook is English".

File: src/contributing/void-docs/style-guide.md (hunk 1 above)
Comment (flexibeast) [reply src/contributing/void-docs/style-guide.md:12]: Good point. Changed to: "Outside of the 'installation' sections, step-by-step instructions ..."

File: src/contributing/void-docs/style-guide.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bobertlo) [reply src/contributing/void-docs/style-guide.md:11]: Maybe 'written language'? Although I may be tempted to submit PRs in verse now ;)

[the remark this answers is not in the document]
Comment (flexibeast) [reply src/contributing/void-docs/style-guide.md:11]: > "Server not found!" I cried. "Can I not reach voidlinux.eu's Plutonian shore?"
> Quoth the Raven, "Nevermore."

More seriously, hmm, how about "written primarily in plain English"?

[the remark this answers is not in the document]
Comment (bobertlo) [reply src/contributing/void-docs/style-guide.md:11]: sure this would work :)

or maybe we could say "written in English (or other languages as translations are added.)"?

Either way is fine, I will leave this up to you.

[the remark this answers is not in the document]
Comment (flexibeast) [reply src/contributing/void-docs/style-guide.md]: Okay, changed to: "... written in English (or the relevant language
in the case of translations of the Handbook)."
